README.md

```diff
@@ -185,6 +185,27 @@ const myRoute = {
 
 > This will use `MyCustomValue` as the view name.
 
+## Using with composition API
+
+In order to use this plugin with composition api (inside your `setup` method), you can just call the custom composable `useGtm`.
+
+Example:
+
+```js
+import {useGtm} from 'vue-gtm';
+
+const gtm = useGtm();
```

Comment:
I'm a little bit confused how this is working.
We have a kinda global variable `gtmPlugin` in line 6, this is initialized in the `install` function.
So first: if you don't call this `install` function it will never be initialized :warning: (?!). So it is more like `let gtmPlugin: GtmPlugin | undefined;` and `export function useGtm(): GtmPlugin | undefined {`. Or am I so wrong and miss anything?
Do other plugins also use something like this `use*` function? I know `createRouter` from https://next.router.vuejs.org/guide/migration/#new-router-becomes-createrouter but I'm completely not into Vue 3 at all cause I'm blocked by Vuetify not supporting Vue 3 right now :disappointed: 

Sorry for all these questions but I just want to learn and make it right in the way that it is concise with other plugins out there so we don't introduce multiple standards
![image](https://imgs.xkcd.com/comics/standards.png)

Reply:
No worries. The way of Vue3 composition API has completely changed the game by putting all of your code inside `setup` method.

But as the code may get bigger and bigger, it's not logical to put hundreds of lines inside this single method (`setup`).

So if you check this [link ](https://v3.vuejs.org/guide/composition-api-introduction.html#standalone-computed-properties) about composition API and how to define a `custom reusable composable` which is simply a reusable block of logic defined inside a function, you can understand why I created this `useGtm` function.

You can notice that examples in the docs contain functions that use naming convention of `use*`.
However all other plugins like `vue-router@next` has its own composable like `useRouter` or `useRoute` and `vue-store` also has `useStore` so the user has access to those objects with the need of using `this`.

Also, inside your own app you should define your own composables like `useStorage` or `useHttp` or etc...

So using this way, I'm not confusing the user about injection and using a custom string of constant and I'm not developing my own standard but I'm just wrapping this local variable `gtmPlugin` with a composable that is exported to be used just like other libraries or plugins.

For your point of initialization, you are totally true. If the user didn't register the plugin so it won't be simply initialized because as I guess, the initialization step is mandatory as the [README.md](https://github.com/mib200/vue-gtm#configuration) says.

Reply:
Brilliant, that's exactly what I needed.
(virtually marked as resolved, but I will let it open so it stays open)